index.html

```diff
@@ -51,7 +51,7 @@ <h3>April 19, 2019</h3>
         </div>
         <div class="register">
             <a href="https://goo.gl/forms/DWhnBaRRVMHm0wTL2" class="nav-register">
```

Comment:
```suggestion
            <a href="#" class="nav-register">
```
Let's just remove the link.